analysis_options.yaml

```diff
@@ -1,2 +1 @@
 analyzer:
```

Comment:
Just delete the file!

Reply:
Ok, if that is what you prefer. (My reasoning is that it isn't harmful to keep, and it is easier to add options later on than to try to remember the file name, etc.)

Reply:
Done

Reply:
Fair – but having an empty file is just confusing for someone reading the code. That's my default...fewer questions for code skimmers

Reply:
Got it